package.json

```diff
@@ -23,7 +23,8 @@
     "gulp-if": "^2.0.2",
     "gulp-uglify": "^3.0.2",
```

Comment:
why is this still here?

Reply:
`gulp-uglify` is still being used, but is configured to use `uglify-es` internally.